core/src/blocktree.rs

```diff
@@ -346,6 +452,21 @@ impl Blocktree {
                 continue;
             }
 
+            #[cfg(feature = "erasure")]
+            {
+                let set_index = ErasureMeta::set_index_for(blob.index());
```

Comment:
Should we check on insert of data blobs for `can_recover()` in case we can recover blobs?

Reply:
Yes.